include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -74,7 +87,7 @@ radix_sort(_ExecutionPolicy&& __exec, _Range&& __rng)
         // TODO: avoid kernel duplication (generate the output storage with the same type as input storage and use swap)
         // TODO: pass _ProcessSize according to DataPerWorkItem
```

Comment:
It might require more time, but what do you think about fulfilling this TODO and passing a desired `DataPerWorkItem` from within `radix_sort_esimd.pass.cpp`?

Reply:
At current moment we have ```DataPerWorkItem``` defined as ```constexpr ::std::uint16_t DataPerWorkItem = 16;```
So we haven't any dependencies at this place now.

Reply:
I see two ways:
 -  either leave a TODO comment,
 -  or modify the test therefore different values of `DataPerWorkItem` are used for single-work-group, cooperative and onesweep implementation. Whereas, it varies for onesweep depending on a data type.

Reply:
It seems there is a suitable TODO comment at (`TODO: pass _ProcessSize according to DataPerWorkItem`)